app/views/backend/base/_state_bar.html.haml

```diff
@@ -10,7 +10,11 @@
       - classes << "active"
       %span{class: classes}= localized_name
     - elsif transition = transitions.detect{|t| t.to == state.name.to_s }
-      = link_to(localized_name, {action: transition.event, id: resource.id}, class: classes, method: :post, data: {confirm: :are_you_sure.tl}, title: transition.event.to_sym.ta)
```

Comment:
@Aquaj ,
I have updated the code and made changes according to you. Please take a look and let me know if any thing missing.
Thanks